images/wordpress/wp-start

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+if test ${WORDPRESS_VERSION}X = latestX; then
+  wp core download --path=/var/www/html
+else
```

Comment:
I wonder if it'd be worth doing a check here to only run `wp core download` if the `WORDPRESS_VERSION` is different from the currently installed version (`wp core version`)? I know it uses the cached local version if it has one, so it's not actually downloading again, but it would reduce the amount of code that needs to run on `docker compose up`.